packages/integrations/cloudflare/README.md

```diff
@@ -191,6 +191,49 @@ export default defineConfig({
 });
 ```
 
+## Wasm module imports
+
+`wasmModuleImports: boolean`
+
+default: `false`
+
+Whether or not to import `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration). 
+
+Add `wasmModuleImports: true` to `astro.config.mjs` to enable in both the Cloudflare build and the Astro dev server. 
+
+```diff
+// astro.config.mjs
+import {defineConfig} from "astro/config";
+import cloudflare from '@astrojs/cloudflare';
+
+export default defineConfig({
+    adapter: cloudflare({
++       wasmModuleImports: true
+    }),
+	  output: 'server'
+})
+```
+
+Once enabled, you can import a web assembly module in Astro with a `.wasm?module` import. 
+
+The following is a simple example of importing a Wasm module that then responds to requests by adding the request's number parameters together. A request to `/add/40/2` would return a response of `42`.
```

Comment:
```suggestion
The following is an example of importing a Wasm module that then responds to requests by adding the request's number parameters together. 
```
Nicely done, but even just this is sufficient! It's just about providing a little context!

Reply:
@sarah11918 thanks! Updated